.sequelizerc

```diff
@@ -0,0 +1,9 @@
+
+const path = require('path');
+
+module.exports = {
+  'config': path.resolve('./src/db/config', 'babelHook.js'),
+  'models-path': path.resolve('src/db', 'models'),
```

Comment:
Models should be in the src folder, not DB folder. We are not using a DB folder.
```suggestion
  'models-path': path.resolve('./src', 'models'),
```